server/aesdsocket.c

```diff
@@ -139,6 +269,7 @@ bool create_daemon()
  */
 void clean()
 {
+    
     close(sockfd);
     close(client_fd);
```

Comment:
What will be true about `client_fd` here when supporting multiple simultaneous client threads?